static/js/message_edit.js

```diff
@@ -158,6 +158,7 @@ function edit_message (row, raw_content) {
          is_editable: is_editable,
          has_been_editable: (editability !== editability_types.NO),
          topic: message.subject,
+         empty_topic : compose.empty_topic_placeholder(),
```

Comment:
I would call this argument empty_topic_placeholder, so that it shows up when someone `git grep`s for it.